src/arc_utilities/timing.cpp

```diff
@@ -83,17 +82,16 @@ void Profiler::startTimer(std::string timer_name)
 bool Profiler::isTimerStarted(std::string timer_name)
 {
     Profiler* m = getInstance();
-    if (m->timers.find(timer_name) == m->timers.end())
-    {
-        return false;
-    }
-    return true;
+    return m->timers.count(timer_name) > 0;
```

Comment:
This is kind of an odd idiom, but I guess it works.

Reply:
std::map does not have a `contains(key)`.
`map::count(key)` is a common alternative. My favorite explanation is `map::count` is a misspelling of `map::contains`.
Usually, I would let the `int` automatically cast to a `bool`, but I put in the explicit `> 0` for @dmcconachie

Reply:
:+1: